Podcast/ExternalProfileViewController.swift

```diff
@@ -131,15 +135,27 @@ class ExternalProfileViewController: ViewController, UITableViewDataSource, UITa
         System.endpointRequestQueue.addOperation(userRequest)
         
         // Now request user subscriptions and favorites
-//        let favoritesRequest = GetUserFavoritesEndpointRequest(userID: id)
-//        favoritesRequest.success = { (favoritesEndpointRequest: EndpointRequest) in
-//            DispatchQueue.main.async {
-//                guard let results = favoritesEndpointRequest.processedResponseValue as? [Episode] else { return }
-//                self.favorites = results
-//                self.profileTableView.reloadData()
-//            }
-//        }
-//        System.endpointRequestQueue.addOperation(favoritesRequest) // UNCOMMENT WHEN FAVORITES ARE DONE
+        let favoritesRequest = FetchUserRecommendationsEndpointRequest(userID: id)
+        favoritesRequest.success = { (favoritesEndpointRequest: EndpointRequest) in
+            DispatchQueue.main.async {
+                guard let results = favoritesEndpointRequest.processedResponseValue as? [Episode] else { return }
```

Comment:
Why are you doing this on another thread? The request has already succeeded .. can you explain your reasoning behind this